src/components/common/TmSelectNetwork.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <SessionFrame>
+    <div class="select-network">
+      <h2 class="session-title">Select a Network</h2>
+      <div class="select-networks-list">
+        <LiSession
```

Comment:
i think we should be re-using the network list and network item components instead of this.

Reply:
Yes, makes a lot of sense